geo_concerns/app/models/image.rb

```diff
@@ -0,0 +1,7 @@
+# Base class for Images that are georeferenced with a bbox
+class Image < ActiveFedora::Base
+  include ::GeoConcerns::ImageBehavior
+  validates_presence_of :title,  message: 'Your work must have a title.'
+  validates_presence_of :georss_box,  message: 'Your work must have a bbox.'
```

Comment:
If we want to enforce bbox's at the Image level, then I'd say to enforce it at the basic geo metadata level. 

Same with title.

Both would keep the code DRY for these `validate_presence_of` calls.